remove exir.capture from coreml example

JacobSzwejbka opened the pull request:
Summary: exir.capture is deprecated we should use export

Differential Revision: D55437992

mergennachin reviewed (approved):
Make sure to rebase, and CI is green, including lintrunner and land please

JacobSzwejbka committed:
remove exir.capture from coreml example (#2724)

Summary:

exir.capture is deprecated we should use export

Reviewed By: mergennachin

Differential Revision: D55437992
